src/lib/ui/app/LoginForm/index.svelte

```diff
@@ -37,6 +40,10 @@
 {:else}
   <Section {title} class="body-2" titleClass="mb-6" {bottomLabel} {bottomAction} {bottomHref}>
     <div class="flex flex-col gap-2">
+      {#if onWalletConnectLogin}
+        <WalletConnect {isSignUp} onLoginComplete={() => onWalletConnectLogin(from)} />
+      {/if}
```

Comment:
1. Let's unify login complete handle, so it could be passed to `WalletConnect` directly as `onLoginComplete`
2. Conditional rendering is redundant